app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
```

Comment:
Consider adding input validation to ensure that the command string contains exactly three elements separated by spaces. This will prevent potential errors when accessing `command_list[1]` and `command_list[2]`.